src/Core/Services/MetadataProviders/PostgreSqlMetadataProvider.cs

```diff
@@ -19,12 +19,13 @@ namespace Azure.DataApiBuilder.Core.Services
     public class PostgreSqlMetadataProvider :
         SqlMetadataProvider<NpgsqlConnection, NpgsqlDataAdapter, NpgsqlCommand>
     {
+
         public PostgreSqlMetadataProvider(
             RuntimeConfigProvider runtimeConfigProvider,
-            IQueryExecutor queryExecutor,
-            IQueryBuilder sqlQueryBuilder,
-            ILogger<ISqlMetadataProvider> logger)
-            : base(runtimeConfigProvider, queryExecutor, sqlQueryBuilder, logger)
+            IAbstractQueryManagerFactory engineFactory,
```

Comment:
nit: rename engineFactory to queryManagerFactory in all the files

Reply:
fixed.